src/Ormolu/Diff/Text.hs

```diff
@@ -235,12 +233,12 @@ toHunks = go 0 False id id []
           piece ->
             if gotChanges
               then
-                let p = reverse bothHistory
-                    currentAcc' = currentAcc . addBothBefore p (piece :)
+                let currentAcc' = currentAcc . addBothBefore p (piece :)
+                    p = reverse bothHistory
                  in go 0 True hunksAcc currentAcc' [] xs
               else
-                let p = reverse (take margin bothHistory)
-                    currentAcc' = addBothBefore p (piece :)
+                let currentAcc' = addBothBefore p (piece :)
+                    p = reverse (take margin bothHistory)
```

Comment:
Just for clarity: There is no actual change here, `p` and `currentAcc'` were just swapped, right?

Reply:
Yes, I just wanted to have `currentAcc'` and `p` defined in the same order everywhere. Could not resist!